Geometry/CaloGeometry/src/FlatHexagon.cc

```diff
@@ -18,6 +18,8 @@ typedef HepGeom::Vector3D<double> DVec3D;
 typedef HepGeom::Plane3D<double> DPlane3D;
 typedef HepGeom::Point3D<double> DPt3D;
 
+static const float tolmin = 1.e-12;
```

Comment:
It is suggested to use `constexpr` for all constant values that can be evaluated at compile time. https://cms-sw.github.io/cms_coding_rules.html